### PR TITLE
working Julia wrapper stub, i.e. it compiles to .so successfully

### PR DESCRIPTION
**Pull request by krachwal**
Created stub library for `PowerModels.jl` and Julia script, which together successfully generate an `.so`.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,373 @@
+Mozilla Public License Version 2.0
```
Comment (davidbrownell) [inline LICENSE:1]: Will this repo be something that they ultimately own? If so, does this match their current license? If not, let's see if Jeff has an opinion on which license to use.

File: PowerModelsLibrary/PowerModelsLibrary.jl [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#=
+PowerModelsLibrary:
+- Julia version: 1.11.1
+- Author: krachwal
```
Comment (davidbrownell) [inline PowerModelsLibrary/PowerModelsLibrary.jl:4]: Is this following an existing convention that they are already using? If not, remove the author and date as it is information that can be retrieved from the git history.

**Review by davidbrownell — APPROVED** [review]

**Review by varun646 — APPROVED** [review]

File: LICENSE (hunk 1 above)
Comment (krachwal) [reply LICENSE:1]: Yes, that is the one they requested.

File: PowerModelsLibrary/PowerModelsLibrary.jl (hunk 2 above)
Comment (krachwal) [reply PowerModelsLibrary/PowerModelsLibrary.jl:4]: The IDE auto-inserted that. I will remove in next version on my current working branch.

**Pull request merged**